lib/config/validation.js

```diff
@@ -6,6 +6,7 @@ const {
   hasValidSchedule,
   hasValidTimezone,
 } = require('../workers/branch/schedule');
+const validator = require('./validation-helpers/managers');
```

Comment:
I think this `const` should probably be named as something like `managerValidator` because in future we might follow this approach of separating each validation into a separate file.

Reply:
Sure. Good point.